cron/internal/config/config.go

```diff
@@ -66,24 +61,20 @@ var (
 
 //nolint:govet
 type config struct {
-	ProjectID              string  `yaml:"project-id"`
-	ResultDataBucketURL    string  `yaml:"result-data-bucket-url"`
-	RequestTopicURL        string  `yaml:"request-topic-url"`
-	RequestSubscriptionURL string  `yaml:"request-subscription-url"`
-	BigQueryDataset        string  `yaml:"bigquery-dataset"`
-	BigQueryTable          string  `yaml:"bigquery-table"`
-	CompletionThreshold    float32 `yaml:"completion-threshold"`
-	WebhookURL             string  `yaml:"webhook-url"`
-	CIIDataBucketURL       string  `yaml:"cii-data-bucket-url"`
-	BlacklistedChecks      string  `yaml:"blacklisted-checks"`
-	MetricExporter         string  `yaml:"metric-exporter"`
-	ShardSize              int     `yaml:"shard-size"`
-	// Raw results.
-	RawResultDataBucketURL string `yaml:"raw-result-data-bucket-url"`
-	RawBigQueryTable       string `yaml:"raw-bigquery-table"`
-	APIResultsBucketURL    string `yaml:"api-results-bucket-url"`
-	InputBucketURL         string `yaml:"input-bucket-url"`
-	InputBucketPrefix      string `yaml:"input-bucket-prefix"`
+	ProjectID              string            `yaml:"project-id"`
+	ResultDataBucketURL    string            `yaml:"result-data-bucket-url"`
+	RequestTopicURL        string            `yaml:"request-topic-url"`
+	RequestSubscriptionURL string            `yaml:"request-subscription-url"`
+	BigQueryDataset        string            `yaml:"bigquery-dataset"`
+	BigQueryTable          string            `yaml:"bigquery-table"`
+	CompletionThreshold    float32           `yaml:"completion-threshold"`
+	WebhookURL             string            `yaml:"webhook-url"`
+	MetricExporter         string            `yaml:"metric-exporter"`
+	ShardSize              int               `yaml:"shard-size"`
+	InputBucketURL         string            `yaml:"input-bucket-url"`
+	InputBucketPrefix      string            `yaml:"input-bucket-prefix"`
+	Criticality            map[string]string `yaml:"criticality"`
+	Scorecard              map[string]string `yaml:"scorecard"`
```

Comment:
This can just be `AdditionalParams map[string]string` ?

Reply:
It could be yes. I think the benefit of having them separate would be the env var prefixes. Something like

```
criticality:
  foo: bar
scorecard:
  foo: bar
```

would be set with `CRITICALITY_FOO` and `SCORECARD_FOO`, respectively. But I could always rename individual fields:

```
optional-params:
  scorecard-foo: bar
  criticality-foo: bar
```

I think something like that could work, but I would need to make changes such that the env name is based entirely off the key, so you end up with  `SCORECARD_FOO` instead of `OPTIONAL_PARAMS_SCORECARD_FOO`. WDUT?

Reply:
A third option is:

```
optional-params:
  scorecard:
    foo: bar
  criticality:
      foo: bar
```
Which would basically just cut out the repetition of the project prefix. I would need to explore how the parsing would happen though.

Reply:
I just realized the env vars for the required params also start with `SCORECARD_`. We should rename these to be a bit more generic. And having `OPTIONAL_PARAMS_SCORECARD_FOO` should be fine IMO.

It might take some refactoring, so let's do this after the criticality pipeline work is unblocked.